File: anagram/anagram.c
#include <stdio.h>
//#include<stdio.h>
#include <stdlib.h>
#include <stdbool.h>
// #include <stdio.h>
#include <string.h>
// Starter Code
bool isAnagram(char str1[10000], char str2[10000])
{
    bool check_string = true;
    // Write your code here without removing the existing code
    // the variable 'str1' and 'str2' contains array of character.
    // modified the variable 'check_string ' contain the output of the program in boolean format.
    return check_string;
}

// Backend Code
int main()
{
    char str1[10000], str2[10000];
    scanf("%s", &str1);
    scanf("%s", &str2);
    if (isAnagram(str1, str2))
    {
        printf("YES");
    }
    else
    {
        printf("NO");
    }

    return 0;
}

File: anagram/anagram.js
var stdin= process.openStdin();
stdin.addListener("data", function(d)
{
var x= d.toString().trim().split("\n");
if(isAnagram(x[0],x[1]))
console.log("YES");
else
console.log("NO");

});

function isAnagram(str1,str2)
{   
    var check_string=true;
    // Write your code here without removing the existing code
    // the variable 'str1' and 'str2' contains array of character.
    // modified the variable 'check_string' contain the output of the program in boolean format.
    return check_string;
};
    